sdks/go/pkg/beam/core/typex/class.go

```diff
@@ -76,6 +78,13 @@ func IsConcrete(t reflect.Type) bool {
 		return false
 	}
 
+	// TODO(wcn): perhaps we need an extension hook to teach this method about
+	// types that are "concrete" because they are already sourced by external code?
+	pt := reflect.TypeOf((*proto.Message)(nil)).Elem()
```

Comment:
Small suggestion: made this a global similarly to other types elsewhere?

Reply:
Done.